src/ophys_etl/pipelines/segment_binarize_pipeline.py

```diff
@@ -0,0 +1,54 @@
+import argschema
+from ophys_etl.transforms.suite2p_wrapper import Suite2PWrapper
+from ophys_etl.transforms.convert_rois import BinarizerAndROICreator
+import tempfile
+import pathlib
+import json
+
+
+class SegmentBinarizeSchema(argschema.ArgSchema):
+    motion_corrected_video = argschema.fields.InputFile(
+        required=True,
+        description="hdf5 motion-correced video")
+    motion_correction_values = argschema.fields.InputFile(
+        required=True,
+        description="motion correction values in a csv")
+
+
+class SegmentBinarize(argschema.ArgSchemaParser):
+    default_schema = SegmentBinarizeSchema
+
+    def run(self):
+        tmpdir = tempfile.TemporaryDirectory()
+        Suite2P_output = pathlib.Path(tmpdir.name) / "Suite2P_output.json"
+
+        # segment with Suite2P, almost all default args
+        segment_args = {
+                "h5py": self.args['motion_corrected_video'],
+                "output_dir": tmpdir.name,
+                "output_json": str(Suite2P_output),
+                "retain_files": ["stat.npy"],
```

Comment:
Is there ever a use case where we would want to retain more than stat.npy, should we parameterize this?

Reply:
Not in production. `stat,npy` in this case is even in a `TMPDIR` and disappears.